process_manager/templates/process_manager/index.html

```diff
@@ -42,9 +42,9 @@
         <div class="card-header">Messages</div>
         <div class="card-body">
           <ul class="list-group">
-            {% if messages %}
-              {% for message in messages %}<li class="list-group-item">{{ message }}</li>{% endfor %}
-            {% endif %}
+            <div hx-get="{% url 'process_manager:messages' %}"
+                 hx-trigger="every 1s"
+                 hx-swap="afterend"></div>
```

Comment:
If I get this right, new messages are added at the end of the current list of messages, without limit. Wouldn't it make more sense - assuming it is possible - to show newest messages on top?

Reply:
Ups, it's doing that already. Didn't realised of the `[::-1]`.